packages/opentelemetry-tracing/src/config.ts

```diff
@@ -30,6 +31,7 @@ export const DEFAULT_MAX_LINKS_PER_SPAN = 32;
  * used to extend the default value.
  */
 export const DEFAULT_CONFIG = {
+  logger: new ConsoleLogger(getEnv().OTEL_LOG_LEVEL),
```

Comment:
should we have ConsoleLogger or rather NoopLogger ?

Reply:
Should be Noop

Reply:
Just to confirm, this will change functionality, by default a ConsoleLogger is used today: https://github.com/open-telemetry/opentelemetry-js/blob/d8b1be8cc9918c76b9063780cd40f857332ef5b0/packages/opentelemetry-tracing/src/BasicTracerProvider.ts#L46

Reply:
I think it is safe to change it to Noop now.

Reply:
I agree

Reply:
Very well.